<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_flight</class>
 <widget class="QDialog" name="Dlg_flight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#bg{background:#1e1e1e}
QWidget#bar{background:#2e2f30}
QToolButton{background:#01ab70}
QToolButton#tb_select:hover{background:#007b4f}
QToolButton#tb_swap:hover{background:#007b4f}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="bg" native="true">
     <widget class="QWidget" name="bar" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>701</width>
        <height>131</height>
       </rect>
      </property>
      <widget class="QComboBox" name="cbb_end">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>81</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lb_icon1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res.qrc">:/res/up.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbm">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>60</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>月</string>
       </property>
      </widget>
      <widget class="QToolButton" name="tb_select">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>40</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string notr="true">选择</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="tb_swap">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>41</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/res/swap.svg</normaloff>:/res/swap.svg</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb_start">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>49</y>
         <width>81</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="lb3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>出发日期</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_icon2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res.qrc">:/res/down.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbd">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>60</y>
         <width>16</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>日</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb1">
       <property name="geometry">
        <rect>
         <x>69</x>
         <y>19</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="text">
        <string>出发</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="text">
        <string>到达</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cbb_day">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="currentText">
        <string>24</string>
       </property>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="cbb_month">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>700</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
